conftest.py

```diff
@@ -32,6 +32,7 @@
 
 
 def pytest_addoption(parser):
+    # pylint: disable=C0301
```

Comment:
How did this end up here? The `pylint` updade should not have impose a new line length limit.